src/components/Modals/ConfirmDeleteModal.jsx

```diff
@@ -14,25 +15,34 @@ const ConfirmDeleteModal = ({
 }) => {
   const { t } = useI18n()
   return (
-    <Modal
-      into="body"
+    <ConfirmDialog
+      open={true}
+      onClose={toggleConfirmDeleteModal}
       title={t('delete-confirmation.title', {
         smart_count: 1
       })}
-      description={t(
+      content={t(
         getConnectedAccounts(contact).length > 0
           ? 'delete-confirmation.description-google'
           : 'delete-confirmation.description-simple',
         {
           smart_count: 1
         }
       )}
-      primaryText={t('delete')}
-      primaryType="danger"
-      primaryAction={() => deleteContact(contact)}
-      secondaryText={t('cancel')}
-      secondaryAction={toggleConfirmDeleteModal}
-      dismissAction={toggleConfirmDeleteModal}
+      actions={
+        <>
+          <Button
+            theme="secondary"
+            label={t('cancel')}
+            onClick={toggleConfirmDeleteModal}
+          />
+          <Button
+            theme="danger"
+            label={t('delete')}
+            onClick={() => deleteContact(contact)}
```

Comment:
Nit: onDeleteContact would make it clearer that it is the parent that manages the deletion of the contact and the hiding of the modal.